SWLOR.CLI/SWLOR.CLI.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
+    <Platforms>AnyCPU;x86</Platforms>
```

Comment:
I don't think edits here are needed/wanted. Same with recipes.tsv. I'd go back and revert those.

Reply:
Yeah - These are the ones I was talking about. Do I revert these on VS and then repush?

Reply:
Either that or just in GitHub itself. You should be able to just copy the text of the old file, edit the file on your branch, paste and save changes. Should work fine.